src/ui/DiffView.cpp

```diff
@@ -804,7 +815,9 @@ class HunkWidget : public QFrame
     mHeader = new Header(diff, patch, index, lfs, submodule, this);
     layout->addWidget(mHeader);
 
-    mEditor = new Editor(this);
+    auto edt = new Editor(this);
+    mEditor = edt;
+    connect(edt, &Editor::aboutToPaint, this, &HunkWidget::load);
```

Comment:
Won't this execute a lot (of times per second, potentially) and thus lead to a slowdown of the application?

Reply:
HunkWidget::load will not really load if loaded before, so I thought it won't lead to a slowdown that is noticeable.
Now I think I have a better idea, we can check whether it is loaded or not before we emit that signal to save some time.